iam-login-service/src/main/java/it/infn/mw/iam/api/client/management/service/DefaultClientManagementService.java

```diff
@@ -139,10 +145,21 @@ public void deleteClientByClientId(String clientId) {
   public void updateClientStatus(String clientId, boolean status, String userId) {
 
     ClientDetailsEntity client = clientService.findClientByClientId(clientId)
-        .orElseThrow(ClientSuppliers.clientNotFound(clientId));
+      .orElseThrow(ClientSuppliers.clientNotFound(clientId));
     client = clientService.updateClientStatus(client, status, userId);
-    String message = "Client " + (status?"enabled":"disabled");
+    String message = "Client " + (status ? "enabled" : "disabled");
     eventPublisher.publishEvent(new ClientStatusChangedEvent(this, client, message));
+    if (!client.getContacts().isEmpty()) {
```

Comment:
What happens if owner's email is also within client's contacts? Probably we could build a set of emails (no duplicates) from client contacts and owners and this can be done through a unique notificationFactory.createClientStatusChangedMessageFor(ClientDetailsEntity client) instead of 2 methods?